build.gradle

```
@@ -19,13 +20,32 @@ dependencies {
 	implementation 'pl.allegro.tech.boot:handlebars-spring-boot-starter:0.3.0'
 	implementation 'org.jgrapht:jgrapht-core:1.0.1'
 	implementation 'io.jsonwebtoken:jjwt:0.9.1'
+	asciidoctor 'org.springframework.restdocs:spring-restdocs-asciidoctor:2.0.4.RELEASE'
+	testImplementation 'org.springframework.restdocs:spring-restdocs-mockmvc:2.0.4.RELEASE'
```

```suggestion
	asciidoctor 'org.springframework.restdocs:spring-restdocs-asciidoctor'
	testImplementation 'org.springframework.restdocs:spring-restdocs-mockmvc'
```

plugins에서 'io.spring.dependency-management'가 어떠한 역할을 하는지 공부해보세요~

---

의존성에 대한 버전을 자동으로 가져오네요. 수정하였습니다.